src/components/textarea/textarea.stories.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { useState } from "react";
+import { ComponentStory } from "@storybook/react";
+
+import CarbonProvider from "../carbon-provider/carbon-provider.component";
+
+import Textarea from ".";
+
+export const DefaultStory: ComponentStory<typeof Textarea> = () => {
+  const [state, setState] = useState("");
+  const setValue = ({ target }: React.ChangeEvent<HTMLInputElement>) => {
+    setState(target.value);
+  };
+  return <Textarea label="Textarea" value={state} onChange={setValue} />;
+};
+DefaultStory.storyName = "default";
```

Comment:
this story is already named in `mdx`, so we don't need that to be declared for second time here.